docs/source/getting-started/preparing-to-install.adoc

```diff
@@ -1,6 +1,6 @@
 include::variables.adoc[]
 
-[[installing-minishift]]
+[[preparing-install-minishift]]
```

Comment:
@robin-owen What about `preparing-minishift-install`

Reply:
@LalatenduMohanty I'm trying to keep the anchors similar to the titles they refer to. Would you prefer that the title of this section be changed generally?

Reply:
@robin-owen Not really, I think I would prefer to keep them similar. Just that it sound grammatically incorrect leaving bad taste in mouth. If you feel strongly that the current form is fine then I can go ahead and merge this.